### PR TITLE
fix: Solana Links

### PR DESCRIPTION
**Pull request by AndonMitev**
## What?

#Notion/ Trello

## Testing?
- [ ] Run tests locally

## Screenshots (optional)
0

### CODE REVIEW

File: src/utils/asset.js [hunk 1]
```diff
@@ -136,7 +136,9 @@ export const getTransactionExplorerLink = (hash, asset, network) => {
 
 export const getAddressExplorerLink = (address, asset, network) => {
   const chain = cryptoassets[asset].chain
-  return `${EXPLORERS[chain][network].address}${address}`
+  const link = `${EXPLORERS[chain][network].address}`
+
+  return link.includes('{address}') ? link.replace('{address}', address) : link + address
```
[the remark this answers is not in the document]
Comment (AndonMitev) [reply src/utils/asset.js]: applied

File: src/utils/asset.js [hunk 2]
```diff
@@ -131,12 +131,14 @@ export const getTransactionExplorerLink = (hash, asset, network) => {
   const chain = cryptoassets[asset].chain
   const link = `${EXPLORERS[chain][network].tx}`
 
-  return link.includes('{hash}') ? link.replace('{hash}', transactionHash) : link + transactionHash
+  return link.includes('{template}') ? link.replace('{template}', transactionHash) : link + transactionHash
```
Comment (monokh) [inline src/utils/asset.js:134]: This is not what i meant 😅 

I mean that it's unnecessary to have 2 ways of adding the hash/address to the URL
The placeholder approach works for everything
so we can add the placeholder `{hash}` to every url in `EXPLORERS`
and then simply do `link.replace('{hash}', transactionHash)`

File: src/utils/asset.js [hunk 3]
```diff
@@ -131,12 +131,14 @@ export const getTransactionExplorerLink = (hash, asset, network) => {
...
 }
 
 export const getAddressExplorerLink = (address, asset, network) => {
   const chain = cryptoassets[asset].chain
-  return `${EXPLORERS[chain][network].address}${address}`
+  const link = `${EXPLORERS[chain][network].address}`
+
+  return link.includes('{template}') ? link.replace('{template}', address) : link + address
```
Comment (monokh) [inline src/utils/asset.js:141]: similar for this one

**Review by monokh — CHANGES_REQUESTED** [review]

File: src/utils/asset.js (hunk 2 above)
Comment (AndonMitev) [reply src/utils/asset.js]: applied

**Review by monokh — APPROVED** [review]

**Pull request merged**